CMakeLists.txt

```diff
@@ -329,12 +337,15 @@ if (GCC_VERSION)
 endif()
 message(STATUS "Configured C_FLAGS: ${CMAKE_C_FLAGS}")
 
-set(LIBRARY_OUTPUT_PATH ${PROJECT_SOURCE_DIR}/${lib_dir})
+set(LIBRARY_OUTPUT_PATH ${dist_dir}/${lib_dir})
 
-set(CMAKE_DEBUG_POSTFIX "_d")
-
-if (CMAKE_BUILD_TYPE MATCHES "Debug")
-    set(CMAKE_EXECUTABLE_SUFFIX "_d${CMAKE_EXECUTABLE_SUFFIX}")
+if (FLATCC_BUILD_INTO_SOURCE_DIR)
```

Comment:
Also let me know if you'd rather that I folded this into the `if (FLATCC_BUILD_INTO_SOURCE_DIR)` on line 171 to consolidate the logic.